config/lumi.py

```diff
@@ -0,0 +1,260 @@
+# Copyright 2016-2022 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# ReFrame CSCS settings
+#
+
+import reframe.utility.osext as osext
+
+project = 'project_462000008'
+
+site_configuration = {
+    'systems': [
+        {
+            'name': 'lumi',
+            'descr': 'LUMI Cray EX Supercomputer',
+            'hostnames': ['ln\d+-nmn', 'uan\d+-nmn.local', '\S+'],
+            'modules_system': 'lmod',
+            'resourcesdir': '/projappl/%s/reframe_resources/' % project,
```

Comment:
```suggestion
            'resourcesdir': f'/projappl/{project}/reframe_resources/',
```
I would use fstrings on this file.

Reply:
Fixed